src/main/java/uk/gov/companieshouse/api/accounts/Kind.java

```diff
@@ -28,7 +28,8 @@
     DIRECTORS_REPORT_STATEMENTS("small-full-accounts-directors-report#statements"),
     DIRECTORS_REPORT_SECRETARY("small-full-accounts-directors-report#secretary"),
     DIRECTORS_REPORT_APPROVAL("small-full-accounts-directors-report#approval"),
-    LOANS_TO_DIRECTORS("small-full-accounts-note#loans-to-directors");
+    LOANS_TO_DIRECTORS("small-full-accounts-note#loans-to-directors"),
+    LOANS_TO_DIRECTORS_LOAN("small-full-accounts-loans-to-directors#loans");
```

Comment:
minor suggestion, should this be LOANS_TO_DIRECTORS_LOANS? to keep with naming conventions?